docs/en/relying-party-instance.rst

```diff
@@ -0,0 +1,385 @@
+.. include:: ../common/common_definitions.rst
+
+.. _relying-party-instance:
+
+Relying Party Instance
+=========================
+
+The Relying Party Instance (RPI) is a mobile or embedded application designed to request, receive, and process Digital Credentials from Wallet Instances in a trusted manner. Each RPI ensures the integrity, confidentiality and authenticity of Credential exchanges, enabling secure interactions between Users and Relying Parties.
+
+There are two primary types of Relying Party Instances, each serving different operational environments:
+
+- **Mobile Relying Party Instance**: a native application running on a mobile device (e.g., smartphone or tablet). Each instance corresponds to a specific installation of the application on a device. A Mobile Relying Party Instance functions as a public client, operating on end-user devices where sensitive credentials cannot be fully protected from potential threats. This implies additional security measures to establish and maintain trust.
+- **Web Relying Party Instance**: a remote application operated by the Relying Party. A Web Relying Party Instance operates as a confidential client, meaning it can securely store confidential configurations (such as its private cryptographic keys) on a remote server. In this context, the Relying Party is unprovisioned and the presentation request and validation is automatically handled by software.
+
+.. note::
+  
+  Unlike the Web Relying Party Instance, a Mobile Relying Party Instance requires proper lifecycle management and special registration procedures managed through the Relying Party Backend.
+
+Further technical and operational details are discussed in the following sections.
+
+
+Mobile Relying Party Instance
+--------------------------------
+
+A Mobile Relying Party Instance is a native application that runs on mobile devices and enables user digital credential verification within the IT-Wallet ecosystem. It operates as a public client on end-user devices, requiring additional security measures to establish trust. It has a well-defined lifecycle with four main states (**Installed**, **Operational**, **Verified**, and **Uninstalled**) and supports essential functionalities such as registration, access certificate reissuance, and revocation. 
```

Comment:
```suggestion
A Mobile Relying Party Instance is a native application that runs on mobile devices and enables User's Digital Credentials verification within the IT-Wallet ecosystem. It operates as a public client on end-user devices, requiring additional security measures to establish trust. It has a well-defined lifecycle with four main states (**Installed**, **Operational**, **Verified**, and **Uninstalled**) and supports essential functionalities such as registration, access certificate reissuance, and revocation. 
```

Reply:
do we have the definition of access certificate?

I believe that `requiring additional security measures to establish trust` is too much generic or painfull, I would only say which security measures it needs.